[Bug] Not Compiling On macOS

After #59, when adding `about` feature of `libcosmic`, the application no longer compiles on macOS.

```rust
error[E0433]: failed to resolve: could not find `PipeFlags` in `pipe`
  --> /Users/vkhitrin/.cargo/git/checkouts/libcosmic-b367e32ffc370f4f/9426a98/src/process.rs:31:67
   |
31 |     let Ok((read, write)) = rustix::pipe::pipe_with(rustix::pipe::PipeFlags::CLOEXEC) else {
   |                                                                   ^^^^^^^^^ could not find `PipeFlags` in `pipe`
   |
note: found an item that was configured out
  --> /Users/vkhitrin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.44/src/pipe.rs:25:31
   |
25 | pub use backend::pipe::types::PipeFlags;
   |                               ^^^^^^^^^
note: the item is gated here
  --> /Users/vkhitrin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.44/src/pipe.rs:24:1
   |
24 | #[cfg(not(apple))]
   | ^^^^^^^^^^^^^^^^^^

error[E0425]: cannot find function `pipe_with` in module `rustix::pipe`
   --> /Users/vkhitrin/.cargo/git/checkouts/libcosmic-b367e32ffc370f4f/9426a98/src/process.rs:31:43
    |
31  |     let Ok((read, write)) = rustix::pipe::pipe_with(rustix::pipe::PipeFlags::CLOEXEC) else {
    |                                           ^^^^^^^^^ not found in `rustix::pipe`
    |
note: found an item that was configured out
   --> /Users/vkhitrin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.44/src/pipe.rs:108:8
    |
108 | pub fn pipe_with(flags: PipeFlags) -> io::Result<(OwnedFd, OwnedFd)> {
    |        ^^^^^^^^^
note: the item is gated here
   --> /Users/vkhitrin/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rustix-0.38.44/src/pipe.rs:99:1
    |
99  | / #[cfg(not(any(
100 | |     apple,
101 | |     target_os = "aix",
102 | |     target_os = "espidf",
103 | |     target_os = "haiku",
104 | |     target_os = "nto"
105 | | )))]
    | |____^
```

--- a/README.md
+++ b/README.md
@@ -25,9 +25,6 @@ cosmicding was tested against linkding releases `1.31.0`, and `1.36.0`.
 
 ## Installation
 
-> [!WARNING]
-> macOS build are currently broken, `about` feature of `libcosmic` uses a dependency which is broken on macOS ([#61](https://github.com/vkhitrin/cosmicding/issues/61)).
-
 > [!NOTE]
 > Currently cosmicding is hard-codded to build Apple Silicon releases for macOS.
 

